.travis.yml

```diff
@@ -17,19 +17,32 @@ matrix:
       dist: xenial
       sudo: required
       compiler: clang
+    - os: linux
+      dist: bionic
```

Comment:
what about focal?

Reply:
Not sure for now, thought that 18.04 LTS used currently more as the new 20.04 LTS

Reply:
why not both?

Reply:
What I do with my side project is build on xenial and focal. I assume if there is a version conflict, it will show up within these bounds

Reply:
Can do with both, let me change :smile: